pydis_site/apps/api/viewsets/bot/pagination.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.pagination import LimitOffsetPagination
```

Comment:
A better place for this file would be `pydis_site/apps/api/pagination.py`. It shouldn't be under `viewsets` since it's not a viewset.